lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -10229,6 +10330,11 @@ void Emit(ParseNode *pnode, ByteCodeGenerator *byteCodeGenerator, FuncInfo *func
             byteCodeGenerator->EmitPropStore(pnode->location, sym, nullptr, funcInfo, true, false);
         }
 
+        if (pnode->sxClass.IsDefaultModuleExport())
```

Comment:
This IsDefaultModuleExport pattern is to handle the case where we don't have a symbol bound for the thing we're trying to export. That only happens for default exports. The class version would look like this:

export default class { };

You can export an anonymous class (or function, generator, async function, etc) under the default export name. However, we still need to know that thing should be stored in module export var storage. So that's what this flag is meant to help us figure out.